src/extension/tsconfigHelper.ts

```diff
@@ -30,17 +31,20 @@ export class TsConfigHelper {
                 return fileSystem.readFile(tsConfigPath, "utf-8");
             })
             .then(function(jsonContents: string): Q.Promise<any> {
-                return JSON.parse(jsonContents);
+                return JSON.parse(stripJsonComments(jsonContents));
             });
     }
 
     /**
      * Writes out a JSON configuration object to the tsconfig.json file.
      */
     public static writeConfigJson(configJson: any): Q.Promise<void> {
+        return;
```

Comment:
We should show a message to the user and explain to him/her, please do <these modifications> in your tsconfig.json file to enable <this feature>